internal/consensus/state_test.go

```diff
@@ -1609,6 +1609,104 @@ func TestStateLock_POLSafety2(t *testing.T) {
 	*/
 	ensureNewProposal(proposalCh, height, round)
 
+	ensurePrevote(voteCh, height, round)
+	validatePrevote(t, cs1, round, vss[0], propBlockID1.Hash)
```

Comment:
Reverted the change here, for when PBTS is disabled.

Reply:
Sorry, I think I don't understand the change. Is the unite test testing with PBTS enabled or disabled?